pre_commit_hooks/loaderon_hooks/tests/general_hooks/check_branch_name_test.py

```diff
@@ -0,0 +1,37 @@
+import subprocess
+import sys
+import uuid
+
+import pytest
+
+from pre_commit_hooks.loaderon_hooks.general_hooks.check_branch_name import main
+
+
+@pytest.fixture(autouse=True)
+def clean_sys_argv():
+    sys.argv = []
+    yield
```

Comment:
the design of the pattern used in `pre-commit-hooks` makes it so you don't mutate or deal with `sys.argv` at all:

```
def main(argv=None):
    parser = argparse.ArgumentParser()
    ...
    parser.parse_args(argv)
    ...

if __name__ == '__main__':
    exit(main())
```

And in the test

```
main(('arg1', 'arg2', 'arg3'))
```

`ArgumentParser.parse_args` knows when `argv=None` that it should use `sys.argv[1:]` automatically